src/client/app/components/csv/MetersCSVUploadComponent.tsx

```diff
@@ -4,107 +4,191 @@
+		}));
+	};
 
-	public render() {
-		const titleStyle: React.CSSProperties = {
-			fontWeight: 'bold',
-			paddingBottom: '5px'
-		};
+	const submitMeters = async (file : File) => {
+		return await uploadCSVApi.submitMeters(meterData, file);
+	};
+
+	const handleClear = () => {
+		setMeterData({
+			meterName: '',
+			gzip: false,
+			headerRow: false,
+			update: false});
+	};
 
-		const checkboxStyle: React.CSSProperties = {
-			paddingBottom: '15px'
-		};
+	const tooltipStyle = {
+		display: 'inline-block',
+		fontSize: '50%',
+		tooltipReadings: 'help.csv.meters'
+	};
 
-		const formStyle: React.CSSProperties = {
-			display: 'flex',
-			justifyContent: 'center',
-			padding: '20px'
-		};
+	const checkBox = {
+		display: 'flex'
+	};
 
-		return (
-			<div style={formStyle}>
-				<Form onSubmit={this.handleSubmit}>
-					<FormFileUploaderComponent formText='csv.upload.meters' reference={this.fileInput} required labelStyle={titleStyle} />
-					<FormGroup check style={checkboxStyle}>
-						<Label>
-							<Input checked={this.props.gzip} type='checkbox' name='gzip' onChange={this.props.toggleGzip} />
-							<FormattedMessage id='csv.common.param.gzip' />
-						</Label>
-					</FormGroup>
-					<FormGroup check style={checkboxStyle}>
-						<Label check>
-							<Input checked={this.props.headerRow} type='checkbox' name='headerRow' onChange={this.props.toggleHeaderRow} />
-							<FormattedMessage id='csv.common.param.header.row' />
-						</Label>
-					</FormGroup>
-					<FormGroup check style={checkboxStyle}>
-						<Label check>
-							<Input checked={this.props.update} type='checkbox' name='update' onChange={this.props.toggleUpdate} />
-							<FormattedMessage id='csv.common.param.update' />
-						</Label>
-					</FormGroup>
-					<FormGroup>
-						<Label style={titleStyle}>
-							<FormattedMessage id='csv.readings.param.meter.name' />
-						</Label>
-						<Input value={this.props.meterName} name='meterName' onChange={this.handleSetMeterName} />
-					</FormGroup>
-					<Button color='secondary' type='submit'>
-						<FormattedMessage id='csv.submit.button' />
-					</Button>
-				</Form>
-			</div>
-		);
-	}
+	return (
+		<Container>
+			<TooltipHelpComponent page='help.csv.header' />
+			<Form onSubmit={handleSubmit}>
+				<Row className="justify-content-md-center">
+					<Col md='auto'>
+						<h2>
+							{translate('csv.upload.meters')}
+							<div style={tooltipStyle}>
+								<TooltipMarkerComponent page='help.csv.header' helpTextId={tooltipStyle.tooltipReadings} />
+							</div>
+						</h2>
+					</Col>
+				</Row>
+				<Row className='justify-content-md-center'>
+					<Col md='auto'>
+						<FormFileUploaderComponent
+							formText='csv.upload.meters'
+							onFileChange={handleFileChange}
+							reference={fileInput}
+							required
+						/>
+						<FormGroup>
+							<Container>
+								<Row>
+									<Col>
+										<Label for='gzip'>
+											<div style={checkBox}>
+												<div>
```

Comment:
Is this div needed? If it is for spacing maybe use the checkBox style to add that.